src/com/lftechnology/java/training/alina/basic/gameleague/Constants.java

```diff
@@ -0,0 +1,8 @@
+package com.lftechnology.java.training.alina.basic.gameleague;
+
+public class Constants {
```

Comment:
instead of defining a Constant class, you could have used Enum and then store points for each enum constant